README.md

```diff
@@ -2,557 +2,87 @@
-    name: Bright Tank 4,
-    cost: 5091,
-    active: TRUE,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### GET Parts /api/v1/resources/:resource_id/parts
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-request: get /api/v1/resources/:resource_id/parts
-response:
-```json
-[
-    {
-        id: 1,
-        name: lever 2000,
-        inventory: 1,
-        active: true,
-        created_at: 2019-07-16T13:15:31.885Z,
-        updated_at: 2019-07-16T13:16:16.146Z
-    },
-    {
-        id: 2,
-        name: Tank patch,
-        inventory: 1,
-        active: true,
-        created_at: 2019-07-16T13:18:09.048Z,
-        updated_at: 2019-07-16T13:18:09.048Z
-    }
-]
-```
-### GET Parts /api/v1/resources/:resource_id/parts/:id
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank patch,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### POST Parts /api/v1/resources/:resource_id/parts
-Request:
-```javascript
-{
-    name: Tank patch,
-    inventory: 1,
-}
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank patch,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### PATCH Parts /api/v1/resources/:resource_id/parts/:id
-Request:
-```javascript
-{
-    name: Tank lid
-}
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank lid,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### GET Tickets /api/v1/resources/:resource_id/tickets
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
+Problem and Solution One
+Currently, Great Divide has a system that is unused and we sought to understand why and how we could design something that would be more likely to be utilized.
+
+Our primary focus for this issue is to make our app mobile friendly since we learned that Great Divide is a place with more people on their feet than there are computer stations and they are very busy doing hands-on work all day long.
+
+Problem and Solution Two
+There are a great many machines and resources at Great Divide that are either unique, extremely mature or both and they require a great deal of maintenance and working knowledge that can sometimes be lost when someone leaves or absent when someone is on vacation.
+
+This problem has a two pronged solution. We created a ticketing system and a notes system both able to tie back to a specific resource or part or both. We have a ticket portal that will be able to display tickets in various ways.
+
+Future functionality aims to predict when maintenance will be needed ideally create a ticket.
```

Comment:
“Future functionality aims to predict when maintenance will be needed ideally create a ticket” maybe should be ==> “Future functionality aims to predict when maintenance will be needed,  ideally creating a ticket automatically.”